app/src/examples/RuntimeTests/tests/withTiming.test.tsx

```diff
@@ -0,0 +1,226 @@
+import { useEffect } from 'react';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  withDelay,
+} from 'react-native-reanimated';
+import React from 'react';
+import { ComparisonMode } from '../ReanimatedRuntimeTestsRunner/types';
+import {
+  describe,
+  test,
+  expect,
+  render,
+  useTestRef,
+  getTestComponent,
+  wait,
+  callTracker,
+  callTrackerFn,
+  getTrackerCallCount,
+} from '../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+const EXAMPLE_COLORS = {
+  coral: [0xff7f50, 'rgb(255,127,80)', '#ff7f50', 'coral'],
+  cornflowerblue: [0x6495ed, 'rgb(100,149,237)', '#6495ed', 'cornflowerblue'],
```

Comment:
I wanted to test some default colors that are less common than "red" or "pink"

Reply:
I often have to change colours when working since some of the most frequently used colours (red, yellow, green etc.) are too bright and it is uncomfortable to look at them on the screen for too long. I don't mean the aesthetics, I just have sensible eyes and bright colours hurt me a little.